### PR TITLE
epic

### PR DESCRIPTION
**Pull request by kambojk**
this is pretty epic

### CODE REVIEW

**Comment by kambojk** [comment]
this happened to me bro

**Review by Brainicism — APPROVED** [review]

**Comment by taahamahdi** [comment]
Can confirm, issue for me too.

File: index.html [hunk 1]
```diff
@@ -15,6 +15,9 @@ <h1> Hello World! I love kpop </h1>
 <a href="http://art.nandi.pw/images/hr/h152.jpg">
     <img src="http://art.nandi.pw/images/hr/h152.jpg" alt="" height="500" />
 </a>
+<a href="https://okthisisepic.s-ul.eu/77JciCry">
+    <img src="https://okthisisepic.s-ul.eu/77JciCry" alt="" height="500" />
```
Comment (abdallaharar) [inline index.html:19]: i see

Comment (Brainicism) [reply index.html:19]: idc

**Comment by taahamahdi** [comment]
he did it

**Comment by taahamahdi** [comment]
I am closing this issue.

**Comment by Brainicism** [comment]
this isn't an issue

**Comment by kambojk** [comment]
im late, wheres the garlic bread

**Comment by Brainicism** [comment]
you arrived too late and it's all gone

**Pull request merged**